viewer/src/main/webapp/viewer-html/components/Edit.js

```diff
@@ -945,6 +946,10 @@ Ext.define("viewer.components.Edit", {
             input.addCls("x-item-disabled");
         }
 
+        input.on({
```

Comment:
Why not add this directly to the options object above?
```
listeners: {
     scope: this,
     change: this.closureOnFormFieldChange
}
```
By using the scope property you do not need to pass `this` as argument and do not need to return a function